link_shortener/commands/update.py

```diff
@@ -71,11 +72,10 @@ async def update_link(request, link_id, data):
 async def reset_password(request, link_id):
     async with request.app.engine.acquire() as conn:
         trans = await conn.begin()
-        query = await conn.execute(links.select().where(
-            links.columns['id'] == link_id
-        ).where(
+        query = await conn.execute(links.select().where(and_(
+            links.columns['id'] == link_id,
             links.columns['password'] != None
```

Comment:
Tady zase moje prasárna, zkus prosím změnit na "is not None" jestli to půjde, nebo zase něco z dokumentace.

Reply:
Fixnute s `inot()`
a68e024